src/pocketmine/MemoryManager.php

```diff
@@ -200,10 +200,7 @@ public function triggerGarbageCollector(){
 		$cycles = gc_collect_cycles();
 		
 		foreach($this->server->getLevels() as $level){
-			if($this->cacheTrigger){
-				$level->clearCache(true);
-			}
```

Comment:
Chunk cache dose not really use a lot of MB, and can help if tons of players join a "lobby" world. Regardless the current cache implementation is crap and chunk cache should be removed by time with a check when the cache was last used instead of clearing the array.